### ISSUE TITLE
No option for no bias feature for Train.readProblem()

### ISSUE THREAD

**Issue by hrzafer** [bwaldvogel/liblinear-java#18]
I have two options to map my `Dataset` object to a liblinear `Problem` object.

1-) Convert programmatically. This is what I do normally. I also add a `bias=1` feature with this method. And it works just fine. 

2-) First save the `Dataset` object as `.libsvm` and then call the `Train.readProblem()`. However,  `readProblem()` adds a bias feature by default. Even if I pass `bias=0`, an extra feature is added for all instances. This is the reason that I can't reproduce the exact same accuracy which I get with the 1st method. I get slightly worse results. 

Here you may recommend not adding the bias value while saving as `.libsvm` since it is already going to be added by the `readProblem()`. But I create training and test sets separately and create two `libsvm` files. Let's imagine a situation like this; the training libsvm file has 100 features and in the test libsvm file, no instance has the 100th feature.  Thus when I call `readProblem()` to read training and test problems, the test problem's dimension is 99 not 100. To prevent this I add the bias feature as the last feature by default. This way both sets has 101 features. 

In conclusion, I think the bias feature must be optional for the `Train.readProblem()` method.

**Comment by bwaldvogel** [comment]
If you use `bias < 0` no bias is added:

> -B bias : if bias >= 0, instance x becomes [x; bias]; if < 0, no bias term added (default -1)
